api/tests/test_user.py

```diff
@@ -1,7 +1,9 @@
 from django.test import TestCase
 from django.contrib.auth import get_user_model
+from rest_framework.test import APIClient
 
 User = get_user_model()
+client = APIClient()
 
 
 class UserTestCase(TestCase):
```

Comment:
Let's also add tests to ensure that only `POST` & `GET` are allowed for this endpoint.

Reply:
on it

Reply:
Looks good.